server/graphql/v2/query/index.js

```diff
@@ -38,6 +39,7 @@ const query = {
   hosts: HostsCollectionQuery,
   individual: IndividualQuery,
   memberInvitations: MemberInvitationsQuery,
+  notifications: NotificationsQuery,
```

Comment:
We'll rename it to 'ActivitySubscriptions'.

Reply:
@kewitz Sorry for the late reply, I don't think we should rename our models, I just want us to take this as an opportunity to introduce a better name, kind of what we need by introducing `Accounts` instead of `Collectives`. `ActivitySubscriptions` looks perfect to me.